integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/schema/management/ElasticsearchIndexSchemaManagerValidationMappingBaseIT.java

```diff
@@ -217,6 +217,28 @@ public void property_attribute_leniency() {
 		setupAndValidate( index );
 	}
 
+	@Test
+	public void floatAndDouble() {
```

Comment:
Adding fields 'float'/'double' type is not enough to reach the validation code of `float`/`double` values: the JSON representation of the schema still doesn't include any `float`/`double` values, so we won't have to validate such values. You need to also add an `indexNullAs` so that we'll have to validate the `null_value` attribute in the schema.